components/script/dom/gamepadlist.rs

```diff
@@ -48,6 +48,15 @@ impl GamepadList {
     pub fn remove_gamepad(&self, index: usize) {
         self.list.borrow_mut().remove(index);
     }
+
+    pub fn list(&self) -> Vec<Option<DomRoot<Gamepad>>> {
```

Comment:
I would say that in a follow-up we can remove gamepadlist, since a `RefCell<Vec<Dom<Gamepad>>>` on the navigator would seem to do the trick.

Reply:
Agreed, I'll probably do that after this and before GamepadHapticActuator